app/src/main/res/layout/photo_details_fragment.xml

```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
@adria88 nos faltaría un `ImageButton` para finalizar el fragment y poder mostrar así lo que había antes (el `ExploreFragment`). Básicamente sería ejecutar lo que ocurre cuando se clica sobre el `BottomNavigation` en la pestaña de `nav_explore`. Idealmente debiera aparecer en un `Toolbar`, donde además se muestre el title de la `Photo`. Como tenemos en el diseño https://ninjamock.com/Designer/Workplace/138968434/Imagenguardada

Sería otro evento, como el de `(activity as MainActivityCommunicator).onPhotoClicked(it)` que habría que notificar a la `MainActivity` indicando algo así `onPhotoDetailsFragmentBackBtnClicked()`. Más adelante, seguramente lo cambiemos y se maneje desde el `MainViewModel` con un observer, pero por ahora nos puede valer así. Te parece @ImanolOlveira ?

Reply:
Esto se debería llevar a cabo usando el stack de navegación, que además es automático. Para eso hay que usar startActivity() o (creo) crear un nuevo stack al incorporar el fragmento.

Reply:
No entiendo muy bien que quieres decir, mañana lo hablamos si tienes un hueco ;)

Reply:
@ImanolOlveira hice pruebas y en estos momentos tenemos 2 pequeños detalles que impiden que funcione correctamente.

1. La pestaña activa, al darle al botón de atrás, sí que cambia el fragmento activo, pero no cambia la pestaña activa en el BottomNavigationView
2. Lo mismo pasa con el FloatingActionButton

Por lo que o bien ignoramos esto o bien investigamos cómo hacerlo o bien modificamos el comportamiento al darle al back (esto lo he realizado anteriormente en una app Android).

Lo podemos discutir todos juntos. Qué os parece si nos reunimos mañana por la tarde?